__test__/lib/util/did.js

```diff
@@ -1,6 +1,6 @@
 const nacl = require('tweetnacl');
 const bs58 = require('bs58');
-const didUtil = require('../../../src/lib/did');
+const didUtil = require('../../../dist/ts-generated/lib/did');
```

Comment:
It might be nice to add an alias, in our tsconfig to the top folder so we don't have to backtrack like this.
```
"paths": {
      "@/*": ["./src/*"]
    }
```

`@/dist/ts-generated/lib/did`

Reply:
Nice thank you!